composer.json

```diff
@@ -14,8 +14,7 @@
     },
     "require-dev" : {
         "squizlabs/php_codesniffer" : "2.9.2",
-        "phpunit/phpunit" : "6.5.*",
-        "phpunit/dbunit": "3.0.*",
+        "phpunit/phpunit" : "^8",
```

Comment:
1. What about the composer.lock file?
2. Are we not using the dbunit package at all? How come we can remove it without any code changes?

Reply:
We are not using the dbunit, we are using the database mock instead.

Reply:
dbunit is abandoned and no longer maintained officially.